README.md

```diff
@@ -1,2 +1,133 @@
-# sparc-request-kubernetes
-Informational repostiory for building and running SPARCRequest for Kubernetes
+# SPARCRequest - Kubernetes
+
+This reference project can be used to inform how your group could run SPARCRequest in a Kubernetes environment. The following is representative how OCTRI chose to implement SPARCRequest.
+
+## Considerations for running SPARCRequest in Kubernetes
+
+### SPARCRequest Customization
+
+By using a second image we are able to modify our instance of SPARCRequest without needing to maintain a separate branch in source control.
+
+See the Image section below for a description how we build and manage our SPARCRequest instance.
+
+### Deployments
+
+We have two deployment workloads defined; one for the main application and a second for the delayed job that performs background tasks like sending emails.
+
+#### Main Deployment
+
+The main SPARCRequest application generally runs well inside Kubernetes, a single process Puma web server running the rails app. As seen in the [`deployment`](./k8s/deployment.yaml) manifest, we specify the resources, configuration, liveness, initContainers and any volumes required for persistence.
+
+The `initContainers` are used to perform database migrations and asset compilation.
```

Comment:
It's worth noting that all of the init containers must run successfully before the workload container is started.

I also completely forgot that I automated the database migrations.